src/main/webapp/scripts/appscript.js

```diff
@@ -74,17 +74,32 @@ async function addConfigToPage() {
 function buildEditForm() {
   const editForm = document.getElementById('edit-config-form');
   editForm.action = buildQueryString();
+  populateEditForm();
   showForm();
 }
 
+/**
+ * Builds a query string specific to the edit form to fetch an entire configuration
+ * @return {string}   contains "vendorID" and "entireConfig", where vendorID is the associated ID and
+ *                    entireConfig is a boolean that should be true, returning the entire config text
+ */
+function buildQueryStringEditForm() {
+  const selectedVendorId = document.getElementById('customer-ids').value;
+  return `/BillingConfig?vendorID=${selectedVendorId}&entireConfig=true`;
+}
+
+/**
+ * Builds a generic query string containing vendor and account IDs
+ * entireConfig=false denotes the response should only contain the specified account
+ */
 function buildQueryString() {
   const selectedVendorId = document.getElementById('customer-ids').value;
   const selectedAccountId = document.getElementById('account-ids').value;
   // check for space in accountID.
-  if(selectedAccountId == "") {
+  if (selectedAccountId == "") {
     window.alert("A vendor ID and account ID have not been set yet!");
   } else {
-    return `/BillingConfig?vendorID=${selectedVendorId}&accountID=${selectedAccountId}`;
+    return `/BillingConfig?vendorID=${selectedVendorId}&accountID=${selectedAccountId}&entireConfig=false`;
```

Comment:
the name `entireConfig` seems a bit confusing to me, but I can't currently think of a better alternative

Reply:
I spent a while thinking about this as well and don't have a good alternative, but I agree that it is awkward